vispy/visuals/__init__.py

```diff
@@ -44,3 +45,4 @@
 from .border import _BorderVisual  # noqa
 from .colorbar import ColorBarVisual  # noqa
 from .graphs import GraphVisual  # noqa
+from .windbarb import WindbarbVisual  # noqa
```

Comment:
Why is the visual imported twice?

Reply:
Not sure, two are better than one? I'll clean-up.